docs/application/dotnet/samples/ui/overview.md

```diff
@@ -99,6 +99,99 @@ for TD:
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="267" src="media/m69linearlayout.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NUI/LinearLayout" target="_blank"><strong>(M) LinearLayout (NUI)</strong></a></p>
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.LinearLayout.html" target="_blank">LinearLayout</a>.</p>
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="267" src="media/m74notification.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NUI/NotificationExample" target="_blank"><strong>(M) Notification (NUI)</strong></a></p>
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Components.Notification.html" target="_blank">Notification</a>.</p>
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="267" src="media/m64pagination.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NUI/Pagination" target="_blank"><strong>(M) Pagination (NUI)</strong></a></p>
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Components.Pagination.html" target="_blank">Pagination</a>.</p>
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="267" src="media/m75scrollablebase.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NUI/ScrollableBaseExample" target="_blank"><strong>(M) ScrollableBase (NUI)</strong></a></p>
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.Components.ScrollableBase.html" target="_blank">ScrollableBase</a>.</p>
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="267" src="media/m63videoview.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NUI/VideoViewSample" target="_blank"><strong>(M) VideoView (NUI)</strong></a></p>
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.VideoView.html" target="_blank">VideoView</a>.<br/>
+			It performs several functionalities like forward, backward, play, pause, or stop a video.</p>
+			</td>
+		</tr>
+		<tr>
+			<td><img alt="" height="267" src="media/m70view.png" width="150"/></td>
+			<td>
+			<p><a href="https://github.com/Samsung/Tizen-CSharp-Samples/tree/master/Mobile/NUI/View" target="_blank"><strong>(M) View (NUI)</strong></a></p>
+			<p>This sample application demonstrates how to use <a href="https://samsung.github.io/TizenFX/latest/api/Tizen.NUI.BaseComponents.View.html" target="_blank">View</a>.<br/>
+			Press **Next** to see different View configurations.</p>
```

Comment:
@sgchoi5 
My intention was to make it bold, I thought md tag will work as it renders html when built, but in this case it's not, so using HTML tag will do it for us. 

```suggestion
			Tap<b>Next</b>to see different View configurations.</p>

Reply:
Thanks! Fixed it!